docs/pipelines/run-cmd.md

```diff
@@ -14,4 +14,54 @@ code and tries to install it (will finish with error if not running from NeMo-Sk
 ns run_cmd --cluster=local cd /nemo_run/code/ && pip install -e .
 ```
 
-There are many more examples of how to use `run_cmd` throughout our documentation.
\ No newline at end of file
+There are many more examples of how to use `run_cmd` throughout our documentation.
+
+## LLM Server and Sandbox Server
+
+While we can run arbitrary commands with the default `run_cmd` script, we also provide the ability to
+run a LLM server with the `--model` argument and a few extra arguments for the server config. These arguments 
+are similar to the ones used for `start_server` script.
+
+This can be useful to run a server on a local machine or on a cluster with GPUs in a slurm job, while also being able to
+run arbirary code that uses LLM calls. 
+
+### Example
+
+Say you have the following inference file that uses OpenAI API with a vLLM backed server (say to run a 
+project that is compatible with OpenAI API). Imagine a file called `inference.py` with the following code:
+
+```python
+from openai import OpenAI
+client = OpenAI(api_key='EMPTY', base_url=f"http://0.0.0.0:5000/v1", timeout=None)
+api_model = client.models.list().data[0].id
+
+response = client.chat.completions.create(
+    model=api_model,
+    messages=[
+        {"role": "user", "content": "What is the capital of France?"},
+    ],
+    temperature=0.7,
+    max_tokens=128,
+    top_p=0.95,
+    n=1,
+    stream=False,
+)
+print(response.choices[0].message.content)
+```
+
+Then we can run the server and the inference code in a single command as below. We will also show how to start the code 
+execution server that can be used to run arbitrary code in a sandboxed environment. This can be useful to safely 
+execute untrusted code or to run code that requires a specific environment in a container without affecting the host 
```

Comment:
I think we need to tone this down, our sandbox is certainly not safe to run untrusted code, and we shouldn't say it in the docs :) It's a little safer than running it without sandbox, but not safe for sure (which we mention here https://nvidia.github.io/NeMo-Skills/basics/sandbox/)

Reply:
Ah ok